dsmr_parser/objects.py

```diff
@@ -183,9 +183,19 @@ def unit(self):
         return self.values[2]['unit']
 
     def __str__(self):
+        timestamp = self.datetime
+        if isinstance(self.datetime, datetime.datetime):
+            timestamp = self.datetime.astimezone().astimezone(pytz.utc).isoformat()
+        timestamp_occurred = self.occurred
+        if isinstance(self.occurred, datetime.datetime):
+            timestamp_occurred = self.occurred.astimezone().astimezone(pytz.utc).isoformat()
+        value = self.value
+        if isinstance(self.value, datetime.datetime):
+            value = self.value.astimezone().astimezone(pytz.utc).isoformat()
+        if isinstance(self.value, Decimal):
+            value = float(self.value)
         output = "{}\t[{}] at {} occurred {}"\
-            .format(str(self.value), str(self.unit), str(self.datetime.astimezone().astimezone(pytz.utc).isoformat()),
-                    str(self.occurred.astimezone().astimezone(pytz.utc).isoformat()))
+            .format(str(value), str(self.unit), str(timestamp), str(timestamp_occurred))
```

Comment:
I suggest using an f-formatted string here

Reply:
f-format is nowhere used in the code atm. Let's keep this for now and maybe do a follow-up PR to replace then all at once?